test/team-management_test.go

```diff
@@ -0,0 +1,38 @@
+package test
+
+import (
+	"encoding/json"
+	"fmt"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+)
+
+func testTeamManagement() {
+	It("should deploy pod with maneki role", func() {
+		stdout, stderr, err := ExecAt(boot0, "kubectl", "run", "-n", "maneki", "neco-ephemeral-test", "--image=quay.io/cybozu/ubuntu-debug:18.04", "pause")
+		Expect(err).NotTo(HaveOccurred(), "stdout: %s, stderr: %s, err: %v", stdout, stderr, err)
+
+		Eventually(func() error {
+			stdout, _, err := ExecAt(boot0, "kubectl", "get", "pod/neco-ephemeral-test", "-o=json")
+			if err != nil {
+				return err
+			}
+			po := new(corev1.Pod)
+			err = json.Unmarshal(stdout, po)
+			Expect(err).NotTo(HaveOccurred())
```

Comment:
Expect should not be used in `Eventually` block.
```suggestion
			if err != nil {
				return nil
			}
```